README.md

```diff
@@ -19,14 +17,14 @@ UI should be now up and running at `http://localhost/`.
 
 More configuration (port numbers, etc.) can be done by editing the `.env` file before `$ make run`.
 
-### Developer Setup
-For setting up a developer environment, Mainflux UI docker image must be built using the following command:
+## Usage
+A developer build from the source can be achieved using the following command:
 ```
 $ git clone https://github.com/mainflux/ui.git
```

Comment:
Please remove these lines if the repeat installation process - mentioning just what `make ui` brings up developer version is sufficient.

Reply:
Should I also remove the second `make run` and also `angular-cli` stuff?